hangman.py

```diff
@@ -0,0 +1,330 @@
+    '''
+    print("You have", guesses_left, "guesses left")
+    get_available_letters(letters_guessed)
+    print("Available letters:", get_available_letters(letters_guessed))
+    guessed_letter = input("Please guess a letter: ")
+    return guessed_letter
+
+
+def not_correct_input(warnings_left, guesses_left, secret_word, letters_guessed):
+    '''
+    System of losing warnings or guesses when letter wasn't input
+    '''
+    if warnings_left <= 0:
+        guesses_left -= 1
+        print("Oops! That is not a valid letter. You have no warnings left so you lose one guess:",
+              get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+    else:
+        warnings_left -= 1
+        print("Oops! That is not a valid letter. You have", warnings_left, "warnings left:",
+              get_guessed_word(secret_word, letters_guessed))
+        return warnings_left
+
+
+def guessed_before(warnings_left, guesses_left, letters_guessed):
+    '''
+    System of losing warnings/guesses for entering letter which was guessed before
+    '''
+    if warnings_left <= 0:
+        guesses_left -= 1
+        print("Oops! You've already guessed that letter. You have no warnings left so you lose one guess:",
+              get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+    else:
+        warnings_left -= 1
+        print("Oops! You've already guessed that letter. You have", warnings_left, "warnings left:",
+              get_guessed_word(secret_word, letters_guessed))
+        return warnings_left
+
+
+def results(guessed_letter, guesses_left, letters_in_secret_word, secret_word, letters_guessed):
+    '''
+    Showing results of current guessing
+    '''
+    if guessed_letter in letters_in_secret_word:
+        # Part of code for correct guess
+        print("Good guess:", get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+    else:
+        # System of losing guesses for incorrect guessed word
+        if guessed_letter in EN_VOWELS:
+            guesses_left -= 2
+        else:
+            guesses_left -= 1
+        print("Oops! That letter is not in my word:", get_guessed_word(secret_word, letters_guessed))
+        return guesses_left
+
+
+def end_of_the_game(winning, guesses_left,secret_word):
+    '''
+    Showing information about game results
+    '''
+    if winning:
+        total_score = guesses_left * len(set(secret_word))
+        print("Congratulations, you won! Your total score for this game is:", total_score)
+    else:
+        print("Sorry, you ran out of guesses. The word was", secret_word)
+
+
+def logical_body_hangman(guesses_left, letters_guessed, warnings_left, letters_in_secret_word, hints_switch):
+    '''
+    This function is the main part of game hangman.
+    Is works with functions during match and showing needed information for the player.
+    It returns false or true, which is understood as loose or win in hangman.
```

Comment:
Could you be consistent in `return` with function name.

Reply:
done